modules/ossm-load-test-results.adoc

```diff
@@ -69,7 +69,7 @@ Every additional filter adds to the path length inside the proxy and affects lat
 
 The Envoy proxy collects raw telemetry data after a response is sent to the client.
 The time spent collecting raw telemetry for a request does not contribute to the total time taken to complete that request.
-However, since the worker is busy handling the request, the worker won't start handling the next request immediately.
+However, since the worker is busy handling the request, the worker will not start handling the next request immediately.
```

Comment:
Per ISG s/since/because
```suggestion
However, because the worker is busy handling the request, the worker does not start handling the next request immediately.
```